outofcore/include/pcl/outofcore/impl/octree_base_node.hpp

```diff
@@ -110,7 +110,7 @@ namespace pcl
       , root_node_ ()
       , parent_ (super)
       , depth_ ()
-      , children_ (std::vector <OutofcoreOctreeBaseNode<ContainerT, PointT>*> (8,static_cast<OutofcoreOctreeBaseNode<ContainerT, PointT>*>(0)))
+      , children_ (std::vector <OutofcoreOctreeBaseNode<ContainerT, PointT>*> (8,static_cast<OutofcoreOctreeBaseNode<ContainerT, PointT>*>(nullptr)))
```

Comment:
Looks crazy. Do you think we can replace this with:

```cpp
, children_ (8, nullptr)
```

(and in similar instances below)